### DISCUSSION TITLE
Beautify Extension Error

### DISCUSSION THREAD

**Discussion by ArunPatal** [ajaxorg/ace#5635] [Q&A] [answered]
Hello,

I am trying to beautify the code but getting error.

```
var format = ace.require("ace/ext/beautify");
format .beautify(session);
```
Error:
` Uncaught TypeError: e.getTokens is not a function`

Above code use to work before but i updated to latest version and now it's not working.
Unfortunately i can not remember the last version where it use to work.

Please help me with this issue.

Thanks

**Comment by nightwing** [comment, marked as answer]
Could you provide an example page demonstrating the issue. The code above works as expected on 
 [this playground example](https://mkslanc.github.io/ace-playground/?value=Ly9zYW1wbGVzL2hlbGxvIHdvcmxkCgovLyBFYXNpZXN0IHdheSB0byBydW4gQUNFIGVkaXRvciBpbiBjb250YWluZXIgd2l0aCBpZD0iZXhhbXBsZSIKdmFyIGVkaXRvciA9IGFjZS5lZGl0KCJleGFtcGxlIiwgewogICAgdGhlbWU6ICJhY2UvdGhlbWUvdGV4dG1hdGUiLAogICAgbW9kZTogImFjZS9tb2RlL2phdmFzY3JpcHQiLAogICAgdmFsdWU6IGBjb25zb2xlLmxvZ1wKICAgICgKICAgICAgICAnSGVsbG8gd29ybGQnCiAgICAgICAgKTtgCn0pOwoKCgpmdW5jdGlvbiBmb3JtYXRTZWxlY3Rpb24oKSB7CiAgICB2YXIgZm9ybWF0ID0gYWNlLnJlcXVpcmUoImFjZS9leHQvYmVhdXRpZnkiKTsKICAgIGZvcm1hdCAuYmVhdXRpZnkoZWRpdG9yLnNlc3Npb24pOwp9CgpcMFwwPHNjcmlwdCBzcmM9Imh0dHBzOi8vd3d3LnVucGtnLmNvbS9hY2UtYnVpbGRzQGxhdGVzdC9zcmMtbm9jb25mbGljdC9hY2UuanMiPjwvc2NyaXB0Pgo8c2NyaXB0IHNyYz0iaHR0cHM6Ly93d3cudW5wa2cuY29tL2FjZS1idWlsZHNAbGF0ZXN0L3NyYy1ub2NvbmZsaWN0L2V4dC1iZWF1dGlmeS5qcyI%2BPC9zY3JpcHQ%2BCjxkaXY%2BPGJ1dHRvbiBvbmNsaWNrPSJmb3JtYXRTZWxlY3Rpb24oKSI%2BRm9ybWF0PC9idXR0b24%2BPC9kaXY%2BCjxkaXYgaWQ9J2V4YW1wbGUnIHN0eWxlPSJoZWlnaHQ6IDEwMHB4Ij48L2Rpdj4%3D#hello-world) of that code working.

**Reply by ArunPatal** [reply]
Am not sure where was the problem. But after cleaning browser history it's working...
